storage/acl/acl_test.go

```diff
@@ -0,0 +1,129 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package acl
+
+import (
+	"bytes"
+	"context"
+	"io/ioutil"
+	"log"
+	"os"
+	"testing"
+	"time"
+
+	"google.golang.org/api/iterator"
+
+	"cloud.google.com/go/storage"
+
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+)
+
+func TestMain(m *testing.M) {
+	// These functions are noisy.
+	log.SetOutput(ioutil.Discard)
+	s := m.Run()
+	log.SetOutput(os.Stderr)
+	os.Exit(s)
+}
+
+// TestAcls runs all of the package tests.
+func TestAcls(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	var (
+		bucket = tc.ProjectID + "-samples-object-bucket-1"
+		object = "foo.txt"
+		buf    bytes.Buffer
+	)
+
+	if err := addBucketACL(bucket); err != nil {
+		t.Errorf("cannot add bucket acl: %v", err)
```

Comment:
Include the name of the function that failed to make this easier to debug when it fails.
```suggestion
		t.Errorf("addBucketACL: %v", err)
```

https://github.com/golang/go/wiki/CodeReviewComments#useful-test-failures